packages/neon-core/src/wallet/Balance.ts

```diff
@@ -100,6 +101,8 @@ export class Balance {
   /** The token balances in this Balance for each token keyed by its symbol. */
   public tokens: { [sym: string]: Fixed8 };
 
+  public tokenHashes: { [sym: string]: string };

```

Comment:
At the moment it looks like the hash is not guaranteed to exists even if the token is present within the balance. Is it possible to guarantee it?

Reply:
when querying the get_balance endpoint in both neoscan and dora, the hash is always present, the "amount" field being greater or equal than zero, and it is also present when the unspent array size is zero or greater than zero. Wouldn't that be the way to guarantee that?